### ISSUE TITLE
[BUG] Crash on Android R with error 'Invalid token LIMIT'.

### ISSUE THREAD

**Issue by debuggerx01** [CaiJingLong/flutter_photo_manager#430]
**Describe the bug**
In a Android R device, it got crash when I want to pick a image.

**To Reproduce**
1. I use [flutter_wechat_assets_picker](https://github.com/fluttercandies/flutter_wechat_assets_picker) in my app.
2. When I use AssetPicker.pickAssets() function, app crashed.

**Smartphone (please complete the following information):**
 - Device: [mi CC9 pro]
 - OS: [Android 11]

**Log**

```bash
E/AndroidRuntime(32229): FATAL EXCEPTION: pool-5-thread-4
E/AndroidRuntime(32229): Process: com.debuggerx.flutterhealthheart, PID: 32229
E/AndroidRuntime(32229): java.lang.IllegalArgumentException: Invalid token LIMIT
E/AndroidRuntime(32229): 	at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:172)
E/AndroidRuntime(32229): 	at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:142)
E/AndroidRuntime(32229): 	at android.content.ContentProviderProxy.query(ContentProviderNative.java:472)
E/AndroidRuntime(32229): 	at android.content.ContentResolver.query(ContentResolver.java:1197)
E/AndroidRuntime(32229): 	at android.content.ContentResolver.query(ContentResolver.java:1128)
E/AndroidRuntime(32229): 	at android.content.ContentResolver.query(ContentResolver.java:1084)
E/AndroidRuntime(32229): 	at top.kikt.imagescanner.core.utils.AndroidQDBUtils.getAssetFromGalleryIdRange(AndroidQDBUtils.kt:188)
E/AndroidRuntime(32229): 	at top.kikt.imagescanner.core.PhotoManager.getAssetListWithRange(PhotoManager.kt:68)
E/AndroidRuntime(32229): 	at top.kikt.imagescanner.core.PhotoManagerPlugin$onHandlePermissionResult$3.invoke(PhotoManagerPlugin.kt:203)
E/AndroidRuntime(32229): 	at top.kikt.imagescanner.core.PhotoManagerPlugin$onHandlePermissionResult$3.invoke(PhotoManagerPlugin.kt:25)
E/AndroidRuntime(32229): 	at top.kikt.imagescanner.core.PhotoManagerPlugin$sam$java_lang_Runnable$0.run(Unknown Source:2)
E/AndroidRuntime(32229): 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
E/AndroidRuntime(32229): 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
E/AndroidRuntime(32229): 	at java.lang.Thread.run(Thread.java:923)
```

maybe related to [not working with Android Q](https://github.com/csdcorp/local_image_provider/issues/28)

**Comment by debuggerx01** [comment]
@CaiJingLong 
I think this issue is caused by this line:
https://github.com/CaiJingLong/flutter_photo_manager/blob/f209651a3db0eae5e31881d4d5b190c5e2f4bbef/android/src/main/kotlin/top/kikt/imagescanner/core/utils/IDBUtils.kt#L321

**Comment by debuggerx01** [comment]
@CaiJingLong 
Solution:
[https://stackoverflow.com/a/62891878](https://stackoverflow.com/a/62891878)

**Comment by debuggerx01** [comment]
Solved by upgrade to v 0.6.0.

**Comment by EquitysoftSagar** [comment]
> Solved by upgrade to v 0.6.0.

which version you upgrade ?
